package.json

```diff
@@ -18,7 +18,7 @@
     "@babel/preset-react": "^7.26.3",
     "@babel/preset-typescript": "^7.27.0",
     "@hypothesis/frontend-build": "^4.0.0",
-    "@hypothesis/frontend-shared": "^9.1.0",
+    "@hypothesis/frontend-shared": "^9.4.0",
```

Comment:
The icon we use for the pending status was introduce in a version newer than 9.1